translation-config.yml

```diff
@@ -1,6 +1,46 @@
+locales:
+  default: en_US
+  supported:
+    - ar
+    - de
+    - en_GB
+    - es
+    - fr_CA
+    - fr_FR
+    - it
+    - ja
+    - nl
+    - pl
+    - pt
+    - ru
+    - sv
+    - zh_CN
+    - zh_HK
+
 bundles:
   - path: .
     extension: properties
     default_locale: en_US
   - path: src/main/resources
     extension: json
+
+snapshot:
+  copy:
+    - en_GB
+
+io:
+  out:
+    generator: translations.extensions.io_xls.XlsExporter
+    name: translations
+    mapping:
+      fr_CA: fr
+      fr_FR: fr
+      en_US: ~
```

Comment:
Why are these two locales assigned `~`. It isn't immediately clear?

Reply:
It means that they would be ignored; that locale will not get exported. I initially thought that defining an `exclude` option to list all locales that would be ignored, but reframed from it just to avoid adding another property. At the time it seemed intuitive to null-out an export as basically ignoring it.